simulation.cpp

```diff
@@ -236,6 +242,12 @@ std::vector<double> new_inputs = create_n_inputs(s.get_env(), get_current_input(
 assign_inputs(s.get_pop(), new_inputs);
 }
 
+void create_inputs(simulation &s)
+{
+  environment &e = s.get_env();
+  e.update_n_inputs(s.get_rng(), s.get_inds_input_size());
```

Comment:
gut, I still feel there is a safer way to do this. Here we are playing with fire a little bit

Reply:
I am not sure I understand why?